### ISSUE TITLE
VS Code crashes when previewing a markdown file a second time

### ISSUE THREAD

**Issue by DragosMiloiuNXP** [microsoft/vscode#191962]
Does this issue occur when all extensions are disabled?: Yes. I uninstalled all the extensions I had.

- VS Code Version: 1.81.1
- OS Version: Windows 10

Steps to Reproduce:

1.  Open VS Code
2.  Open a markdown file
3. Right click on the md file and select "Open Preview"
4. Search "ggg" in the preview window (this is mandatory in order to reproduce the issue)
5. Close the preview window
6. Right click on the md file and select "Open Preview" again. -> VS Code exits

**Comment by DragosMiloiuNXP** [comment]
Additional comment: Opening any webview after step 5 also results in a VS Code crash.

**Comment by mgleria-govplus** [comment]
Happening also on:

- VS Code Version: 1.84.2
- OS Version: macOS Sonoma 14.0 (23A344)

It doesn't happen the first time I open the preview, but when I switch from any other code tab to the preview one.

**Comment by mjbvz** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/177046 based on the symptoms
